### ISSUE TITLE
Request for Training Details on Inpainting Task

### ISSUE THREAD

**Issue by Russell-Zeng** [xyfJASON/ctrlora#17]
Thank you for sharing such excellent work with the community! I’m currently exploring the inpainting task and would appreciate more details on the training process, specifically how the training data is constructed and the input format used for the model. Any additional information would be greatly appreciated!

**Comment by xyfJASON** [comment]
Thanks for your interest in our work! As detailed [here](https://github.com/xyfJASON/ctrlora/blob/main/README-traineval.md), we use the COCO2017 dataset to train our model for the inpainting task. The masks are generated with the [BrushInpainter](https://github.com/xyfJASON/ctrlora/blob/5b5c26bae7daeeab523b682a88364a5042b3f959/annotator/inpainting_brush/__init__.py) annotator, which draws random brush-like masks on the input image. The masked areas are set to black, i.e., the pixel values are 0. During training, the masked images are [normalized](https://github.com/xyfJASON/ctrlora/blob/5b5c26bae7daeeab523b682a88364a5042b3f959/datasets/custom_dataset.py#L74) to [0,1] and then fed into the model as the condition.

**Comment by xyfJASON** [comment]
“The masked images” refer to the images where the inpainting areas are masked out from the ground truth.
